internal/store/ulimit.go

```diff
@@ -18,47 +18,43 @@
 package store
 
 import (
-	"fmt"
 	"os"
-	"os/exec"
 	"runtime"
-	"strconv"
-	"strings"
+
+	"golang.org/x/sys/unix"
 )
 
-func uLimit() int {
+func isFdCloseToULimit() bool {
 	if runtime.GOOS != "darwin" && runtime.GOOS != "linux" {
-		return 0
+		return false
 	}
-	out, err := exec.Command("bash", "-c", "ulimit -n").Output()
-	if err != nil {
-		log.Print(err)
-		return 0
+
+	var fdPath string
+	switch runtime.GOOS {
+	case "darwin":
+		fdPath = "/dev/fd"
+	case "linux":
+		fdPath = "/proc/self/fd"
 	}
-	outStr := strings.Trim(string(out), " \n")
-	num, err := strconv.Atoi(outStr)
+	f, err := os.Open(fdPath)
 	if err != nil {
-		log.Print(err)
-		return 0
-	}
-	return num
-}
-
-func isFdCloseToULimit() bool {
-	if runtime.GOOS != "darwin" && runtime.GOOS != "linux" {
+		log.Warn("isFdCloseToULimit: ", err)
 		return false
 	}
-
-	pid := fmt.Sprint(os.Getpid())
-	out, err := exec.Command("lsof", "-p", pid).Output() //nolint[gosec]
+	d, err := f.ReadDir(-1)
 	if err != nil {
 		log.Warn("isFdCloseToULimit: ", err)
 		return false
 	}
```

Comment:
This section could be far more simply expressed as 

```
	d, err := os.ReadDir(fdPath)
	if err != nil {
		log.Warn("isFdCloseToULimit: ", err)
		return false
	}
```
but for whatever reason the bridge is built with go1.15 (a now unsupported version of the build chain/stdlib by 2 releases).